src/datachain/lib/file.py

```diff
@@ -115,22 +115,24 @@ class File(DataModel):
     parent: str = Field(default="")
     name: str
     size: int = Field(default=0)
-    version: str = Field(default="")
     etag: str = Field(default="")
+    version: str = Field(default="")
     is_latest: bool = Field(default=True)
-    last_modified: datetime = Field(default=TIME_ZERO)
-    location: Optional[Union[dict, list[dict]]] = Field(default=None)
     vtype: str = Field(default="")
+    location: Optional[Union[dict, list[dict]]] = Field(default=None)
+    last_modified: datetime = Field(default=TIME_ZERO)
```

Comment:
Changing the order of fields is (to some extent) a compatibility break. I wouldn't change it unless we have a good reason.

Reply:
I would not worry about compatibility yet. Also, we are on zero-ver, so breakages are expected.

Anyway, I'll change the `_datachain_column_types` order to keep them in sync.